init_data.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import logging
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from data import split_data, PTCL_HEADER, PTCL_FEATURES
+from util import fix_seed
+
+
+def main(args):
+    logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s', level=logging.INFO)
+
+    fix_seed(args.seed)
```

Comment:
Move closer to split_data or use sklearn

Reply:
use sklearn